readme.md

```diff
@@ -120,7 +120,7 @@ $ ava --help
   test.js test-*.js test/*.js
 ```
 
-Files in directories named `fixtures` and `helpers` are ignored, as well as files starting with `_`. This can be useful for having helpers in the same directory as your test files.
+Directories are recursive by default. Files in directories named `fixtures` and `helpers` are ignored, as well as files starting with `_`. This can be useful for having helpers in the same directory as your test files.
```

Comment:
@sindresorhus: But that doesn't really indicate that any directories that you pass it, nor does it include the bit about fixtures and helpers or underscores.